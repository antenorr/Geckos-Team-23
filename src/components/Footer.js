import React from 'react';

const Footer = () => (
    <div className="footer"><a href="https://chingu.io/">Courtesy of Chingu Developers Network</a></div>
);

export default Footer;