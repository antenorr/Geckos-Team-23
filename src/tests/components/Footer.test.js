import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../../components/Footer';

test('should render Footer correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});