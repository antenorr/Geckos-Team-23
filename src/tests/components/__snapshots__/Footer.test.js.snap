// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<div
  className="footer"
>
  <a
    href="https://chingu.io/"
  >
    Courtesy of Chingu Developers Network
  </a>
</div>
`;
